packages/fxa-payments-server/src/routes/Product/index.test.tsx

```diff
@@ -260,13 +264,15 @@ describe('routes/Product', () => {
       getByTestId,
       findByText,
       queryByText,
+      matchMedia,
       navigateToUrl,
       apiMocks,
     } = await commonSubmitSetup(createToken);
 
     fireEvent.click(getByTestId('submit'));
 
     await findByText('Your subscription is ready');
+    expect(matchMedia).toBeCalled();
```

Comment:
This should probably be something like `expect(matchMedia).toBeCalledWith("(max-width: 480px)");` - or better yet, a constant imported from the component. 

Should also have a couple cases to mock `matchMedia` with both `true` & `false` results to exercise the `lineHeight` change in `checkMedia`